app/adaptive_tessellation/LoggerDataCollector.h

```diff
@@ -0,0 +1,189 @@
+#pragma once
+
+#include <spdlog/stopwatch.h>
+#include "AdaptiveTessellation.h"
+#include "wmtk/utils/getRSS.h"
+
+namespace adaptive_tessellation {
+
+class LoggerDataCollector
+{
+    class StatisticsObj
```

Comment:
we should deprecate defining classes in classes, but probably not worth doing for now (this statistics object seems to be an internal detail anyway so it doesn't need to be defined either)